src/screens/AdminPortal/ManageTag/ManageTag.module.css

```
@@ -147,3 +233,267 @@
+  border-radius: var(--radius-xl);
+}
+
+.manageMode [class~='modal-title'] {
+  color: var(--color-black);
+  font-family: Lato, sans-serif;
+  font-size: var(--font-size-2xl);
+  font-weight: var(--font-weight-bold);
+  line-height: var(--line-height-tight);
+}
+
+:global(.modal-header .btn-danger) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)